engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -73,6 +73,16 @@ object Runtime {
       notAppliedArguments: Vector[Int]
     )
 
+    /** The type of the expression.
+      *
+      * @param visibleType the public type of the expression visible to the user
+      * @param conversionType the available conversions
+      */
+    case class ExpressionType(
+      visibleType: Vector[String],
```

Comment:
I am not sure I like the terminology:
- maybe we use _visible type_ somewhere
- but we certainly never used **conversion type**!

The [documentation](https://github.com/enso-org/enso/blob/b0d05a367a3d3b86c43694efff93761586c002fb/docs/types/intersection-types.md#narrowing-type-check) is using following terms:
- _"has been cast to (the visible part)"_
- _"hidden types .... can be cast to."_

I understand that the current terminology isn't perfect. We can change it, but consistently. We shouldn't be inventing `conversionType` term in the protocol only. If we want new term, we should update (at least) docs as well.